apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -279,7 +284,7 @@
                     }
                 },
                 "debug" : {
-                    "description" : "Whether to enable the debug endpoints",
+                    "description" : "deprecated",
```

Comment:
Could you please add a test config file (`apps/aeutils/test/data`) hooked to unit tests showing that the presence of this config does not make the validation of the config fail? (I did not find one.)

Reply:
👍